app/src/main/res/values/strings.xml

```diff
@@ -373,6 +373,7 @@
     <string name="MUSIC_TURN_REPEAT_ALL_ON">Turn Repeat All On</string>
     <string name="MUSIC_TURN_REPEAT_ONE_ON">Turn Repeat One On</string>
     <string name="MUSIC_TURN_REPEAT_OFF">Turn Repeat Off</string>
+    <string name="MUSIC_TEMPORARY_PLAYLIST_DESCRIPTION">AndroidAutoIDrive temporary playlist</string>
```

Comment:
I'm planning on renaming the project soon as part of 1.3.1, can this be pre-emptively updated to AAIdrive?

Reply:
Yeah sure thing, updated this as well as the corresponding string from L.kt